bindings/python/SimplifiedModelControllers/tests/test_centroidal_mpc.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import bipedal_locomotion_framework.bindings as blf
+import numpy as np
+import manifpy as manif
+from datetime import timedelta
+
+pytestmark = pytest.mark.mpc
+
+def test_centroidal_mpc():
+
+    centroidal_mpc = blf.reduced_model_controllers.CentroidalMPC()
```

Comment:
Why the CentroidalMPC here? Should this be a test on the CoMZMP controller instead?

Reply:
It's a mistake :(

Reply:
I fixed it adding the correct test